webui/react/src/pages/DesignKit.tsx

```diff
@@ -316,6 +316,7 @@ const ChartsSection: React.FC = () => {
     },
   };
   const [xAxis, setXAxis] = useState<XAxisDomain>(XAxisDomain.Batches);
+  const [scale, setScale] = useState<Scale>(Scale.Linear);
```

Comment:
A good pattern here is to provide a `useLineChart` hook that manages the data in cases where the parent component doesn't need to handle it (like in UI Kit or simpler charts).